example-dvc-experiments/generate.bash

```diff
@@ -136,11 +136,21 @@ git tag "baseline-experiment"
 
 dvc exp run -n cnn-32 --queue -S model.conv_units=32
 dvc exp run -n cnn-64 --queue -S model.conv_units=64
-dvc exp run -n cnn-96 --queue -S model.conv_units=96
 dvc exp run -n cnn-128 --queue -S model.conv_units=128
 
 dvc exp run --run-all --jobs 2
 
+
+tag_tick
+cp -f ${HERE}/code-dvclive/train.py src/train.py
+cp -f ${HERE}/code-dvclive/requirements.txt requirements.txt
+git add src/train.py requirements.txt
+git commit -m "DVCLive modifications"
```

Comment:
The `dvc.yaml` (or `dvc stage add` command) should be updated accordingly to use `live` field instead of `plots` and `metrics`.

Reply:
I'm updating all steps to contain DVCLive.